packages/flutter/lib/src/material/animated_icons/animated_icons.dart

```diff
@@ -284,7 +284,11 @@ class _PathClose extends _PathCommand {
 // not be smooth enough we can try applying spline instead.
 //
 // [progress] is expected to be between 0.0 and 1.0.
```

Comment:
Side question: is it intentional to have simple comments instead of doc-comments?

Reply:
Yeah, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use--for-public-quality-private-documentation

Reply:
Thanks for the link. I need to read it again since it seems to have received several changed since I read it.